docs/conf.py

```diff
@@ -358,6 +358,47 @@
 # texinfo_no_detailmenu = False
 
 
+# -- Options for autodoc --------------------------------------------------
+
+
+def autodoc_skip_member_handler(app, what, name, obj, skip, options):
+    """
+    Skips members from internal modules (like _cryptography_rsa or base)
```

Comment:
Please could you file a bug for issue that requires this docs workaround, even if it will be closed with the changes in this PR.  Add a link to the issue in this comment.